infra/terraform/global/variables.tf

```diff
@@ -11,3 +11,5 @@ variable "es_password" {}
 variable "es_scheme" {
   default = "https"
 }
+
+variable "uploads_bucket_name" {}
```

Comment:
We may have a default value here. Unless we think the bucket name itself is somehow sensitive (don't think so).